src/components/pool/DepositTab.tsx

```diff
@@ -83,6 +83,8 @@ export const SectionLabel = styled.div`
   color: rgba(130, 160, 182, 1);
 `;
 
+const TOOLTIP_CONTENT_DEPOSIT = 'Deposit amounts are based on current prices. If prices shift while your transaction is pending, some funds may be returned to you instead of being deposited to Blend. Slippage is the threshold between returning funds and cancelling the transaction altogether (to save gas)';
```

Comment:
Just a heads up, cancelling is the spelling for the non-American variant. I usually prefer to use 'canceling' instead, but feel free to use whatever you like, just thought I'd point it out just in case it was a mistake.

Reply:
Also, don't forget a period at the end.